assets/dev/js/frontend/handlers/video.js

```diff
@@ -63,12 +59,22 @@ export default class Video extends elementorModules.frontend.handlers.Base {
 		}
 	}
 
-	animateVideo() {
-		this.getLightBox().setEntranceAnimation( this.getCurrentDeviceSetting( 'lightbox_content_animation' ) );
+	async animateVideo() {
+		const lightbox = await elementorFrontend.utils.lightbox;
```

Comment:
It should come from the assets loader, not from the utils.

Reply:
@matipojo - The asset loader handles external libraries, the lightbox is a module chunk, and in order for the lightbox to work properly it also handles the dialog and share-link dynamic loading. Therefore, the lightbox util handles everything that is needed for the lightbox to work properly.